github/store-github-release.sh

```diff
@@ -24,8 +24,24 @@ ASSETS=$(find "outputs/" -type f)
 
 echo $ASSETS
 
+# Example RELEASE_URL - 'https://github.com/balena-io-hardware/hardware-process-test/releases/tag/untagged-0b077a1f62efcb13db09'
 resinci-deploy store github-release "${ASSETS}" \
   -b ${headBranch} \
   -r ${repo} \
   -o ${org} \
-  -v ${version}
+  -v ${version} > output.txt
+
+cat output.txt
+RELEASE_URL=$( grep -nri -A 10 'Published draft release:' output.txt | grep html_url -A 1 | grep -e "[\'\"]" )
+
+echo $RELEASE_URL
```

Comment:
How about a more descriptive log, like "Sending email with subject blah to blah" or whatever you think is valuable.